sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileScanRDD.scala

```diff
@@ -38,7 +38,7 @@ import org.apache.spark.util.NextIterator
  * that need to be prepended to each row.
  *
  * @param partitionValues value of partition columns to be prepended to each row.
- * @param filePath path of the file to read
```

Comment:
nit: Maybe add a comment here that we throw an exception if were passed an invalid URI.

Reply:
We don't throw an exception in the constructor. It depends on how downstream uses it. Validating that it is a valid URI is beyond the scope of this PR.